apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerConnectionStringBuilder.cs

```diff
@@ -88,6 +89,36 @@ public string CredentialFile
             set => this[CredentialFileKeyword] = value;
         }
 
+        /// <summary>
+        /// Option to change between the default handling of null database values (return <see cref="DBNull.Value">DBNull.Value</see>) or
+        /// the legacy handling (return the default value for whatever type is requested).
```

Comment:
We decided not to use the word "legacy" in the connection-string option name; so it might be better not to use it here either?
Possibly "non-standard"?

Reply:
I wondered about that - non-standard sounds fine to me. Will update in another commit.